src/app/(main)/my-voyage/[teamId]/ideation/components/ContributionCard.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+
+import { useParams } from "next/navigation";
+import Link from "next/link";
+import { useEffect, useState } from "react";
+import Badge from "@/components/badge/Badge";
+import Button from "@/components/Button";
+import { VoyageMember } from "@/store/features/ideation/ideationSlice";
+import { useAppSelector } from "@/store/hooks";
+import { cn } from "@/lib/utils";
+import routePaths from "@/utils/routePaths";
+
+interface ContributionCardProps {
+  contributed_by: {
+    member: VoyageMember;
+  };
+  projectIdeaId: number;
+  className?: string;
+}
+
+function ContributionCard({
+  contributed_by,
+  className,
+  projectIdeaId,
+}: ContributionCardProps) {
+  const { teamId } = useParams<{ teamId: string }>();
+  const { id } = useAppSelector((state) => state.user);
+  const [ownVote, setOwnVote] = useState(false);
+
+  useEffect(() => {
+    if (contributed_by.member.id === id) {
+      setOwnVote(true);
+    }
```

Comment:
I'm not really too picky on things like this. I think it's just largely personal preference and no real benefit to either or. I do prefer to just put the braces because it's less typing if the thing being run needs to be extended. Another big reason is, I use a snippet for this lol